docs/src/getting-started/getting-linuxcnc.txt

```diff
@@ -111,9 +103,13 @@ a computer.  The image is too large to fit on a CD.
   then use this command:
 +
 -----
-dd if=linuxcnc-2.7-wheezy.iso of=/dev/sde
+dd if=linuxcnc-2.7-stretch.iso of=/dev/sde
 -----
 
+The https://www.debian.org/CD/faq/#write-usb[Debian FAQ] provides more
+information about how to work with USB media, including how to use
+win32diskimager to write the image to USB media on Microsoft Windows.
```

Comment:
It might be worth mentioning https://etcher.io/ here. It is by far the easiest to use tool for writing boot-able images to USB, and is open source and cross platform.